datalad/tests/test_dataset.py

```diff
@@ -83,3 +95,84 @@ def test_Dataset_get(src, dst):
     assert_equal(f.readlines(), ['123\n'], "test-annex.dat's content doesn't match.")
 
     os.chdir(cwd)
+
+
+@assert_cwd_unchanged
+@with_testrepos(flavors=local_flavors)
+@with_tempfile
+def test_Dataset_add_to_annex(src, dst):
+
+    ds = Dataset(dst, src)
+    filename = 'file_to_annex.dat'
+    filename_abs = os.path.join(dst, filename)
+    f = open(filename_abs, 'w')
+    f.write("What to write?")
+    f.close()
```

Comment:
too common of a pattern, and since we don't really care about content here, we could create a tests helper e.g.

```python
def create_file(path, content="default load"):
     ...
```
to be used in such points -- will leave to you